### PR TITLE
Release notes for release_1.0.2

### PR DESCRIPTION
**Pull request by wenlingz**
Signed-off-by: wenlingz <wenling.zhang@intel.com>

### CODE REVIEW

**Comment by deb-intel** [comment]
@NanlinXie  Do we have customers who use v1.0?? Why are we creating a v1.02 version?

File: doc/release_notes/release_notes_1.0.2.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+ACRN is a flexible, lightweight reference hypervisor that's built with
+real-time and safety-criticality in mind and is optimized to streamline
+embedded development through an open source platform.  Check out
+:ref:`introduction` for more information. All project ACRN source code
+is maintained in the https://github.com/projectacrn/acrn-hypervisor
+repository and includes folders for the ACRN hypervisor, the ACRN device
+model, tools, and documentation. You can either download this source
+code as a zip or tar.gz file (see the `ACRN v1.0.2 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.0.1>`_)
```
Comment (NanlinXie) [inline doc/release_notes/release_notes_1.0.2.rst]: Should be "https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.0.2"

File: doc/release_notes/release_notes_1.0.2.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+   $ git clone https://github.com/projectacrn/acrn-hypervisor
+   $ cd acrn-hypervisor
+   $ git checkout v1.0.2
+
+There were no documentation changes in this update, so you can still
+refer to the v1.0-specific documentation found at
+https://projectacrn.github.io/1.0/.
+
+Change Log in version 1.0.2 since version 1.0
```
Comment (NanlinXie) [inline doc/release_notes/release_notes_1.0.2.rst]: Should be 1.0.1

File: doc/release_notes/release_notes_1.0.2.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+*********************************************
+
+Primary changes are to fix several security and stability issues found
+on the v1.0 branch, as listed here:
```
Comment (NanlinXie) [inline doc/release_notes/release_notes_1.0.2.rst]: Should be 1.0.1

**Comment by NanlinXie** [comment]
@deb-intel , v1.0 release branch is our major release and we will keep maintaining it with security fix.

File: doc/release_notes/release_notes_1.0.2.rst (hunk 1 above)
Comment (wenlingz) [reply doc/release_notes/release_notes_1.0.2.rst]: done

**Comment by NanlinXie** [comment]
@deb-intel , Please help to check and merge this PR. Thanks!

**Pull request merged**